### PR TITLE
`P` commands: move `help_msg_*[]` arrays to top

### PR DESCRIPTION
**Pull request by MaskRay**
Press `P Tab`

```
% r2 -e cmd.comphint=1 --
|Usage: P[?osi] [file]Project management
| Pc [file]    show project script to console
| Pd [file]    delete project
| Pi [file]    show project information
| Pl           list all projects
| Pn[j]        show project notes (Pnj for json)
| Pn [base64]  set notes text
| Pn -         edit notes with cfg.editor
| Po [file]    open project
| Ps [file]    save project
| PS [file]    save script file
| P- [file]    delete project (alias for Pd)
| NOTE:        See 'e??prj.'
| NOTE:        project are stored in ~/.config/radare2/projects
[0x00000000]> P
```

### CODE REVIEW

File: libr/core/cmd.c [hunk 1]
```diff
@@ -3160,6 +3160,7 @@ R_API void r_core_cmd_init(RCore *core) {
 	r_cmd_add (core->rcmd, "write",    "write bytes", &cmd_write);
 	r_cmd_add (core->rcmd, "Code",     "code metadata", &cmd_meta);
 	r_cmd_add (core->rcmd, "Project",  "project", &cmd_project);
+	cmd_project_init ();
```
Comment (radare) [inline libr/core/cmd.c]: Why here?

File: libr/core/cmd_project.c [hunk 2]
```diff
@@ -4,6 +4,42 @@
...
+	"Pn-", "str", "delete lines matching /str/ in notes",
+	"Pn+", "str", "append one line to the notes",
+	"Pnj", "", "show notes in base64",
+	"Pnj", " [base64]", "set notes in base64",
+	"Pnx", "", "run project note commands",
+	NULL
+};
+
+static void cmd_project_init(void) {
+	DEFINE_CMD_DESCRIPTOR(P);
```
Comment (radare) [inline libr/core/cmd_project.c]: Missing space before (

Comment (MaskRay) [inline libr/core/cmd_project.c]: Done

File: libr/core/cmd.c (hunk 1 above)
Comment (MaskRay) [inline libr/core/cmd.c]: No need to resolve merge conflicts..

**Pull request merged**